<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saleDetail</class>
 <widget class="QWidget" name="saleDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>724</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1276</width>
    <height>724</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1276</width>
    <height>724</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1276</width>
     <height>724</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{border-image: url(:/Picture/saledetail.jpg);border-top-left-radius:15px;border-top-right-radius:15px;border-bottom-left-radius:15px;border-bottom-right-radius:15px;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>640</y>
      <width>370</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;padding:2px 4px;color: white;</string>
    </property>
    <property name="text">
     <string>Back to Manager Page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>291</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>500</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255,255,255);border: 3px solid white;border-radius:8px;color:rgb(74,112,133);</string>
    </property>
    <property name="text">
     <string>导出表格</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>451</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>620</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Picture/arrowRight.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_info">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>590</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="set_info">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>570</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
